spec/rails_helper.rb

```diff
@@ -90,6 +90,20 @@ def call(env)
   # instead of true.
   config.use_transactional_fixtures = true
 
+  if ENV['DONT_REUSE_INITIAL_FABRICATION']
```

Comment:
nice, go with `FABRICATION_REUSE=0` and just default to 1

Reply:
Ok